main.go

```diff
@@ -150,6 +150,10 @@ func main() {
 			"pkiCACertFile":     pkiCaFile,
 			"ccdFile":           serverCcdFile,
 		}}
+	// delete any existing secrets present
+	_ = clientset.CoreV1().Secrets(namespace).Delete(context.TODO(), clientSecret.Name, v1.DeleteOptions{})
```

Comment:
don't we need to check if it exists or not?

Reply:
Not really required